pkg/netmap/policy.go

```diff
@@ -11,3 +11,24 @@ type Filter = netmap.Filter
 type Replica = netmap.Replica
 type Clause = netmap.Clause
 type Operation = netmap.Operation
+
+func PlacementPolicyToJSON(p *PlacementPolicy) ([]byte, error) {
+	return netmap.PlacementPolicyToJSON(p)
+}
+
+func PlacementPolicyFromJSON(data []byte) (*PlacementPolicy, error) {
+	return netmap.PlacementPolicyFromJSON(data)
+}
+
+func NodeFromJSON(data []byte) (*Node, error) {
+	ni, err := netmap.NodeInfoFromJSON(data)
+	if err != nil {
+		return nil, err
+	}
+
+	return newNodeV2(0, ni), nil
```

Comment:
Node always has index that is associated with position in network map. Therefore is it correct to parse single node out of network map context? Not sure.

Reply:
I think its not correct, because `Node` is a special type for working with network map, not an API message.

Reply:
Removed JSON converter for SDK's node structure for now.